src/main/java/carpet/settings/ParsedRule.java

```diff
@@ -63,7 +63,7 @@
         }
         else if (this.type == boolean.class || (this.type == String.class && categories.contains(RuleCategory.COMMAND)))
```

Comment:
if its a boolean type, then do not add 'ops'. Add ops only if its string class and command

Reply:
done